cmd/kubernetes-discovery/pkg/cmd/server/start.go

```diff
@@ -61,9 +61,10 @@ func NewCommandStartDiscoveryServer(out, err io.Writer) *cobra.Command {
 		StdOut: out,
 		StdErr: err,
 	}
+	o.Etcd.StorageConfig.Type = storagebackend.StorageTypeETCD3
 	o.Etcd.StorageConfig.Prefix = defaultEtcdPathPrefix
 	o.Etcd.StorageConfig.Codec = api.Codecs.LegacyCodec(v1alpha1.SchemeGroupVersion)
-	o.SecureServing.ServingOptions.BindPort = 9090
+	o.SecureServing.ServingOptions.BindPort = 443
```

Comment:
> Sure about the default port? would better choose something like 7443, in a analogy to 6443.

Yeah.  I want this very pod centric. I'm hoping to make it really easy (well, as easy as it can be), to launch in a pod.

I might even work on having default locations for the various certs, so you just have to mount the secrets.